go.mod

```diff
@@ -25,19 +25,19 @@ require (
 	github.com/mcuadros/go-version v0.0.0-20180611085657-6d5863ca60fa
 	github.com/morikuni/aec v0.0.0-20170113033406-39771216ff4c // indirect
 	github.com/pkg/errors v0.8.1
-	github.com/rancher/norman v0.0.0-20200312033725-5c74e1ee1e6d
-	github.com/rancher/types v0.0.0-20200326224903-b4612bd96d9b
+	github.com/rancher/norman v0.0.0-20200326201949-eb806263e8ad
```

Comment:
Need to update this after merging https://github.com/rancher/norman/pull/364 with the commit from the release/v2.4 branch